src/gui/plugins/resource_spawner/ResourceSpawner.qrc

```diff
@@ -1,8 +1,10 @@
 <!DOCTYPE RCC><RCC version="1.0">
 <qresource prefix="ResourceSpawner/">
+  <file>ResourceSpawner.qml</file>
```

Comment:
Let's keep these in alphabetic order, it helps finding things later

Reply:
Resolved fdff6c8149c682c4141e5110e3b31d8618730867